godel_robots/abb/godel_irb2400/godel_irb2400_support/urdf/abb_blending_eff_macro.xacro

```diff
@@ -1,100 +1,154 @@
+
+    <!-- End Effector "Body" Geometry -->
     <link name="${prefix}gripper_body">
       <visual>
         <geometry>
-          <mesh filename="package://godel_robot_resources/meshes/visual/blending_eff.stl" />
+          <mesh filename="package://godel_irb2400_support/meshes/visual/Bracket.STL"/>
+          <!--<mesh filename="package://godel_irb2400_support/meshes/visual/ensenso_blending_eff.stl"/>-->
         </geometry>
         <material name="grey">
           <color rgba="0.5 0.5 0.5 1.0"/>
         </material>
       </visual>
       <collision>
-       <geometry>
-          <mesh filename="package://godel_robot_resources/meshes/collision/blending_eff.stl" />
+        <geometry>
+          <mesh filename="package://godel_irb2400_support/meshes/collision/ensenso_blending_eff.stl"/>
         </geometry>
       </collision>
     </link>
 
-    <!-- Adjust these values after calibration -->
-    <link name="${prefix}kinect2_mount"/>
-    
-    <!-- Conditionally include the 'calibratable' camera definition -->
-    <xacro:if value="${calibration}">
-      <xacro:calib_kinect2_sensor_definitions prefix="${prefix}" parent_link="${prefix}kinect2_mount"/>
-    </xacro:if>
-    
-    <xacro:unless value="${calibration}">
-      <xacro:kinect2_sensor_definitions prefix="${prefix}" parent_link="${prefix}kinect2_mount" 
-        x_val="-0.05490207720545535" 
-        y_val="-0.1108059263096891" 
-        z_val="-0.01737111616208457" 
-        roll="1.591199900154373" 
-        pitch="0.009672124637678808" 
-        yaw="-1.573233882247353"/>
-    </xacro:unless>
-
-    <joint name="${prefix}gripper_to_keyence_optical" type="fixed">
-      <parent link="${prefix}gripper_body"/>
-      <child link="${prefix}keyence_sensor_optical_frame"/>
-      <origin xyz="${keyence_optical_x} ${keyence_optical_y} ${keyence_optical_z}" rpy="0 -1.5707963267948966 0"/>
+    <joint name="${prefix}eff_frame_to_eff_body" type="fixed">
+      <parent link="${prefix}eff_frame"/>
+      <child link="${prefix}gripper_body"/>
     </joint>
 
-    <joint name="${prefix}keyence_optical_to_keyence_tcp" type="fixed">
-      <parent link="${prefix}keyence_sensor_optical_frame"/>
-      <child link="${prefix}keyence_tcp_frame"/>
-      <origin rpy="0 3.141592653589793 0"/>
+    <link name="${prefix}blending_tool">
+        <visual>
+            <geometry>
+                <mesh filename="package://godel_irb2400_support/meshes/visual/Blending_Tool.STL"/>
+            </geometry>
+            <material name="red">
+                <color rgba="1 0 0 1"/>
+            </material>
+        </visual>
+    </link>
+
+    <joint name="${prefix}bracket_to_blending_tool" type="fixed">
+        <parent link="${prefix}gripper_body"/>
+        <child link="${prefix}blending_tool"/>
     </joint>
 
-    <joint name="${prefix}gripper_to_palm" type="fixed">
-      <parent link="${prefix}eff_frame"/>
-      <child link="${prefix}gripper_body"/>
+    <link name="${prefix}ensenso">
```

Comment:
We should make an effort to document the XACRO level interface that a Godel application requires and put it somewhere. A lot of this is my fault, but I'm no longer sure what needs an `asus` frame vs a `ensenso` frame etc.